paimon-core/src/main/java/org/apache/paimon/partition/actions/PartitionMarkDoneAction.java

```diff
@@ -29,36 +30,62 @@
 
 import static org.apache.paimon.CoreOptions.METASTORE_PARTITIONED_TABLE;
 import static org.apache.paimon.CoreOptions.PARTITION_MARK_DONE_ACTION;
+import static org.apache.paimon.CoreOptions.PARTITION_MARK_DONE_CUSTOM_CLASS;
 import static org.apache.paimon.utils.Preconditions.checkArgument;
 import static org.apache.paimon.utils.Preconditions.checkNotNull;
 
 /** Action to mark partitions done. */
 public interface PartitionMarkDoneAction extends Closeable {
 
+    String SUCCESS_FILE = "success-file";
```

Comment:
public

Reply:
All fields in an interface are implicitly public.

Reply:
ok